dts/bindings/pinctrl/nuvoton,npcm-pinctrl.yaml

```diff
@@ -0,0 +1,154 @@
+        The alternate pin selection configurations consist of a total of 6 groups.
+        The group order is from 0 to 5, and the bit N value in each group is for
+        register N's bit value.
+        Set NPCM_PIN_CFG_NULL for an empty configuration.
+    nuvoton,id:
+      type: array
+      description: |
+        The pinmux group definition property.
+        The pin's index. The name of this property follows the group's order.
+    nuvoton,low-voltage-reg:
+      type: array
+      description: |
+        The low-voltage level definition property.
+        The low-voltage level register and its bit.
+    nuvoton,low-voltage-def:
+      type: array
+      description: |
+        The low-voltage level definition property.
+        The default voltage level in hardware of supported alt functions. Set with either
+        NPCM_IO_VOLT_LV_1V8 or NPCM_IO_VOLT_LV_3V3.
+    nuvoton,low-voltage-sup:
+      type: array
+      description: |
+        The low-voltage level definition property.
+        The flag of low-voltage level support. The bit N value represents that group N supports
+        low-voltage level.
+    nuvoton,pull-up-down-config-extend:
+      type: boolean
+      description: |
+        The peripheral pull-up/down definition property.
+        Indicates that there are two groups of peripheral pull-up/down configurations. The default
+        group number is 1.
+    nuvoton,pull-up-down-reg:
+      type: array
+      description: |
+        The peripheral pull-up/down definition property.
+        The peripheral pull-up/down register and its bit. There are a total of 2 registers if
+        property pull-up-down-config-extend is set.
+    nuvoton,pull-up-down-sup:
+      type: array
+      description: |
+        The peripheral pull-up/down definition property.
+        The flag of peripheral pull-up/down support. The bit N value represents that group N
+        supports peripheral pull-up/down level. The maximum number of supported groups is 2.
+
+  child-binding:
+    description: |
+      The NPCM pin controller pin configuration group node is responsible for setting pin
+      configurations including bias-pull-up, bias-pull-down, pinmux alternate function selection,
+      voltage level, and device control.
+
+    include:
+      - name: pincfg-node.yaml
+        property-allowlist:
+          - bias-pull-down
+          - bias-pull-up
+
+    properties:
+      pinmux:
+        type: array
+        required: true
+        description: |
+          The pin configuration group property.
+          The alternate pin group index and selection of alternate function.
+      nuvoton,device-control:
+        type: array
+        description: |
+          The pin configuration group property.
+          Configurations of the device control such as tri-state, IO type, and so on.
+      nuvoton,voltage-level-mv:
+        type: int
+        description: |
+          The pin configuration group property.
```

Comment:
descriptions are misleading, please provide a more concise description/examples in the binding docs.

Reply:
Done.